project/dimensions/enduses.csv

```diff
@@ -31,3 +28,14 @@ electricity_chp,Combined Heat and Power,electricity,MWh
 electricity_thermal_dg,Thermal DG,electricity,MWh
 electricity_dpv,Distributed PV,electricity,MWh
 electricity_unspecified,Unspecified,electricity,MWh
+electricity_ev_ldv_home_l1,ev_ldv_home_l1,electricity,kWh
+electricity_ev_ldv_home_l2,ev_ldv_home_l2,electricity,kWh
+electricity_ev_ldv_work_l1,ev_ldv_work_l1,electricity,kWh
+electricity_ev_ldv_work_l2,ev_ldv_work_l2,electricity,kWh
+electricity_ev_ldv_public_l2,ev_ldv_public_l2,electricity,kWh
+electricity_ev_ldv_public_dcfc,ev_ldv_public_dcfc,electricity,kWh
+electricity_ev_ldv_enroute_dcfc,ev_ldv_enroute_dcfc,electricity,kWh
+electricity_ev_mhdv_depot_ac,ev_mhdv_depot_ac,electricity,kWh
+electricity_ev_mhdv_depot_dc,ev_mhdv_depot_dc,electricity,kWh
+electricity_ev_mhdv_opportunity_dc,ev_mhdv_opportunity_dc,electricity,kWh
+electricity_ev_mhdv_enroute_dc,ev_mhdv_enroute_dc,electricity,kWh
```

Comment:
Please put these above Electric Transit Rail. (Mostly I think it's most readable for electricity_unspecified to be the last entry in this file.)